lib/rex/ui/text/dispatcher_shell.rb

```diff
@@ -657,7 +658,8 @@ def help_to_s(opts = {})
     str = ''
 
     dispatcher_stack.reverse.each { |dispatcher|
-      str << dispatcher.help_to_s
+      max_command_length = dispatcher.commands.map { |command| command[0].length }.max
```

Comment:
I think the goal is for the name colummn to be visually aligned across all of the dispatchers - so this needs to be calculated outside of the loop